src/ResourceManager/Sql/Commands.Sql/help/Get-AzSqlInstanceKeyVaultKey.md

```diff
@@ -1,44 +1,43 @@
 ---
-external help file: Microsoft.Azure.Commands.Sql.dll-Help.xml
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Sql.dll-Help.xml
 Module Name: Az.Sql
-online version: https://docs.microsoft.com/en-us/powershell/module/Az.sql/Get-AzSqlManagedInstanceKeyVaultKey
+online version: https://docs.microsoft.com/en-us/powershell/module/Az.sql/Get-AzSqlInstanceKeyVaultKey
 schema: 2.0.0
 ---
 
-# Get-AzSqlManagedInstanceKeyVaultKey
+# Get-AzSqlInstanceKeyVaultKey
 
 ## SYNOPSIS
-Gets a SQL managed instance's Key Vault keys.
+{{Fill in the Synopsis}}
```

Comment:
Please make sure all these are filled in.

Reply:
Thanks, it looks this was the only place